lambdas/user_access_patterns/lambda_for_createUser/index.js

```diff
@@ -1,6 +1,58 @@
-const AWS = require('aws-sdk');
-exports.handler = async (event) => {
-    // TODO implement
-    const response = (success, reason, data) => { return { success: success, reason: reason, data: data } }
-    return response(true, null, null);
+const AWS = require("aws-sdk");
+const crypto = require("crypto");
+//AWS.config.loadFromPath("../../../keys.json");
+//Use the below code for local setup
+AWS.config.update({
+  region: "local",
+  endpoint: "http://localhost:8000",
+});
+
+const dynamo = new AWS.DynamoDB.DocumentClient();
+function response(statusCode, message) {
```

Comment:
you have decided to use this response template throughout the lambdas??

Reply:
nothing sure of it, but its convinient. Somewhere modification to the template might be necessary.

Reply:
please talk to other members and finalize this, 
@chhavituteja what do you say??

Reply:
@lakshya-20 , confirm this.